explainaboard/metrics/text_to_sql_test.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for explainaboard.metrics.text_to_sql
+
+We test the two metrics with examples in
+intergration_tests.metric_test.test_sql_exactsetmatch
+and intergration_tests.metric_test.test_sql_execution.
+The test requires network connection.
+"""
+
+from __future__ import annotations
+
+import unittest
+
+from explainaboard.metrics.text_to_sql import (
+    SQLExactSetMatch,
+    SQLExactSetMatchConfig,
+    SQLExecution,
+    SQLExecutionConfig,
+)
+
+
+class SQLExactSetMatchConfigTest(unittest.TestCase):
+    def test_to_metric(self) -> None:
+        self.assertIsInstance(
+            SQLExactSetMatchConfig("Accuracy").to_metric(), SQLExactSetMatch
```

Comment:
It was hard for me to understand what "Accuracy" stood for here. Could you add the argument name as well just so it's a little more clear? Same below.

Reply:
I think it was an obsolete implementation. I will remove the string "Accuracy" here.